compiler/cpp/src/thrift/generate/t_rs_generator.cc

```diff
@@ -414,6 +414,9 @@ class t_rs_generator : public t_generator {
   // Return a string representing the rift `protocol::TType` given a `t_type`.
   string to_rust_field_type_enum(t_type* ttype);
 
+  // Return a string representing the `const` rust type given a `t_type`
+  string to_rust_const_type(t_type* ttype, bool ordered_float = true);
```

Comment:
Good call. Makes sense to split this out from the `to_rust_type`. Could you please put this function declaration right before `to_rust_type`? And do the same for implementation please? It just ensures that code that has similar functionality is in the same part of the file.